projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -5880,6 +5925,45 @@ describe('IgxGrid - Filtering actions - Excel style filtering #grid', () => {
                 { ID: 10, Name: { FirstName: 'Eduardo', LastName: 'Ramirez' }, JobTitle: 'Manager', Company: 'Company E' }]);
         }));
     });
+
+    describe('External Excel Style Filtering', () => {
+        let fix, grid;
+        beforeEach(fakeAsync(() => {
+            fix = TestBed.createComponent(IgxGridExternalESFComponent);
+            grid = fix.componentInstance.grid;
+            fix.detectChanges();
+        }));
+
+        it('Should allow hosting Excel Style filtering component outside of the grid.', fakeAsync(() => {
+            // sort
+            const excelMenu = fix.nativeElement.querySelector('.igx-excel-filter__menu');
+            const sortComponent = excelMenu.querySelector('.igx-excel-filter__sort');
+            const sortAsc = sortComponent.lastElementChild.children[0].children[0];
+            sortAsc.click();
```

Comment:
Click logic can be replaced with helper method: **clickSortAscInExcelStyleFiltering**

Reply:
Done.